Makefile

```diff
@@ -189,7 +199,7 @@ docker-build-machinepoollet: ## Build machinepoollet image.
 
 .PHONY: docker-build-machinebroker
 docker-build-machinebroker: ## Build machinebroker image.
-	docker build --target machinebroker -t ${MACHINEBROKER_IMG} .
+	docker build --build-arg LDFLAGS="-X $(MACHINEBROKER_VERSION)=$(VERSION) -X $(MACHINEBROKER_COMMIT)=$(COMMIT)" --target machinebroker -t ${MACHINEBROKER_IMG} .
```

Comment:
I suggest introducing a default value for `LDFLAGS` which provides `-s -w` if not set, e.g. 
```
LDFLAGS ?= "-s -w"
```